src/hypothesis/settings.py

```diff
@@ -261,11 +263,46 @@ def __exit__(self, *args, **kwargs):
         default_context_manager = self.defaults_stack().pop()
         return default_context_manager.__exit__(*args, **kwargs)
 
-    default = DefaultSettings()
+    @staticmethod
+    def register_profile(name, settings):
+        """
+        registers a collection of values to be used as a
```

Comment:
Docstring now slightly strange because of the move from using kwargs to a Settings object.